testutil/mockrelayer/account/mocks.go

```diff
@@ -0,0 +1,11 @@
+package mockaccount
```

Comment:
Same as above comments I think adopting the pattern used in #187, we can avoid mocking cosmos-sdk components all together (we just need their interfaces)